### PR TITLE
Feature/registration comps

### PR DESCRIPTION
**Pull request by jbped**
- Fleshed out the mobile verification portion of the join page. Minimal styling, will need to do a second pass on that. 
- Removed existing mobile number form from the registration page
- Renamed the component from verification-form to mobile-verification

### CODE REVIEW

File: src/app/components/pages/registration/registration.component.html [hunk 1]
```diff
@@ -58,15 +58,8 @@
...
-            <input type="text"
-              placeholder="Enter/select your address"
-              aria-label="Address"
-              matInput
-              appAutocompleteForceSelection
-              formControlName="address"
-              [matAutocomplete]="autoAddress"
-              required
-            >
+            <input type="text" placeholder="Enter/select your address" aria-label="Address" matInput
```
Comment (tipjoe) [inline src/app/components/pages/registration/registration.component.html:61]: We can leave this for now but when an element has a bunch of properties I prefer the longer form above for readability.

File: src/assets/images/le-poo.svg [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (tipjoe) [inline src/assets/images/le-poo.svg:1]: LOL, le-poo

**Review by tipjoe — APPROVED** [review]
Looks good. One thing we've ignored until now is unit tests. Let's talk more about it next week.

**Comment by tipjoe** [comment]
You can resolve the conversations above... just comments.

File: src/assets/images/le-poo.svg (hunk 2 above)
Comment (jbped) [reply src/assets/images/le-poo.svg:1]: 😁

File: src/app/components/pages/registration/registration.component.html (hunk 1 above)
Comment (jbped) [reply src/app/components/pages/registration/registration.component.html:61]: I didn't change anything here, how did this get edited?

**Pull request merged**